core/equations.gms

```diff
@@ -623,6 +623,26 @@ q_emiMac(t,regi,emiMac) ..
   )
 ;
 
+***--------------------------------------------------
+*' All CDR emissions summed up
+***--------------------------------------------------
+q_emiCdrAll(t,regi)..
+  vm_emiCdrAll(t,regi)
+       =e= !! BECC + DACC
+  (sum(emiBECCS2te(enty,enty2,te,enty3),vm_emiTeDetail(t,regi,enty,enty2,te,enty3))
+  + sum(teCCS2rlf(te,rlf), vm_ccs_cdr(t,regi,"cco2","ico2","ccsinje",rlf)))
+  !! scaled by the fraction that gets stored geologically
+  * (sum(teCCS2rlf(te,rlf),
+        vm_co2CCS(t,regi,"cco2","ico2",te,rlf)) /
+  (sum(teCCS2rlf(te,rlf),
+        vm_co2capture(t,regi,"cco2","ico2","ccsinje",rlf))+sm_eps))
+  !! net negative emissions from co2luc
+  -  p_macBaseMagpieNegCo2(t,regi)
+       !! negative emissions from the cdr module that are not stored geologically
```

Comment:
I am not sure whether that is correct as ``vm_ccs_cdr`` refers to the total carbon *captured* by DAC, not the stored carbon, right?

I would think ``pm_share_CCS_CCO2 * vm_ccs_cdr`` would give you the DACCS.